crates/ruma-common/src/events/secret_storage/key.rs

```diff
@@ -0,0 +1,121 @@
+//! Types for the *m.secret_storage.key.\** event.
```

Comment:
For consistency with the current style of docs for other events, please change to

```suggestion
//! Types for the [`m.secret_storage.key.*`] event.
```

plus a spec link.